code/lib/util.py

```diff
@@ -20,15 +20,57 @@
 from rasterio import MemoryFile, mask, windows
 # from scipy.ndimage import convolve
 # import osr
+import re
```

Comment:
what is this package?

Reply:
Python regular expression package used to interpret the inputs of the user and extract the coordinates of the powerplants